app/jobs/in_person/enrollments_ready_for_status_check/error_reporter.rb

```diff
@@ -0,0 +1,29 @@
+module InPerson
+  module EnrollmentsReadyForStatusCheck
+    class ErrorReporter
+      # @param [String] class_name Class for which to report errors
+      # @param [Analytics] analytics
+      def initialize(class_name, analytics)
+        @class_name = class_name
+        @analytics = analytics
+      end
+
+      # Reports an error. A non-StandardError will be converted to
+      # a RuntimeError before being reported.
+      # @param [#to_s,StandardError] error
+      def report_error(error, **extra)
+        error = RuntimeError.new("#{@class_name}: #{error}") unless error.is_a?(StandardError)
+        analytics.idv_in_person_proofing_enrollments_ready_for_status_check_job_ingestion_error(
+          exception_class: error.class,
+          exception_message: error.message,
+          **extra,
+        )
+        NewRelic::Agent.notice_error(error)
```

Comment:
My vote is to remove this direct NewRelic notification, we tend to react to errors in NR as if they are unhandled 500s so seeing errors like this that were caught might make something seem more urgent than it is

Reply:
This kind of error signals that USPS is sending us garbage data or that AWS has become severely misconfigured. At the current scale it may not be consequential, but it could be extremely consequential at the scales we were preparing for earlier this year.

Reply:
I'm fine keeping the NR notification, but we should reconsider if it turns out not to be useful